index.html

```diff
@@ -174,95 +173,117 @@
+          <li>Try to <a>validate</a> <var>urlA</var> and <var>urlB</var>. If
+          either throws, return false.
           </li>
-          <li>The identifiers match if <em>urlA</em> <strong>equals</strong>
-          <em>urlB</em> using the <a>URL equivalence</a> test (i.e. the test
-          returns true).
+          <li>Return <var>urlA</var> <a>equals</a> <var>urlB</var>.
           </li>
-        </ul>
+        </ol>
       </section>
     </section>
     <section>
       <h2>
-        W3C-Defined Strings as Payment Method Identifiers
+        Standardized Payment Method Identifiers
       </h2>
       <p>
-        W3C expects to publish a small number of payment method specifications
-        that define abstractions across similar payment methods (such as credit
-        card payments, tokenized payments, or credit transfers). W3C expects
-        these payment methods will be implemented in a wide variety of third
-        party payment apps. W3C identifies these payment methods with (short)
-        strings; W3C expects to mint these strings in payment method
-        specifications published by the Web Payments Working Group.
+        The <a href="https://w3.org">W3C</a> expects to publish a small number
+        of payment method specifications that define abstractions across
+        similar payment methods (such as credit card payments, tokenized
+        payments, or credit transfers). The <a href="https://w3.org">W3C</a>
+        expects these payment methods will be implemented in a wide variety of
+        third party payment handlers. The W3C identifies these payment methods
+        with (short) strings (individual referred to as a <dfn>standardized
+        payment method identifier</dfn>); the W3C expects to mint these strings
+        in payment method specifications published by the <a href=
+        "https://www.w3.org/Payments/WG/">Web Payments Working Group</a>.
       </p>
       <section>
         <h2>
           Syntax
         </h2>
         <p>
-          The syntax of W3C strings used as payment method identifiers is
-          constrained by this regular expression:
+          The syntax of <a>standardized payment method identifier</a> used as
+          payment method identifiers is constrained by this regular expression:
         </p>
         <pre>
           [a-z0-9-]+
         </pre>
       </section>
       <section>
         <h2>
-          Matching
+          Comparison
         </h2>
         <p>
-          For W3C strings, user agents MUST use exact string matching.
+          For <a>standardized payment method identifier</a>, user agents MUST
+          compare strings in a <a data-cite=
+          "HTML#case-sensitive">case-sensitive</a> manner (code point for code
+          point).
         </p>
       </section>
     </section>
     <section>
       <h2>
+        Security consideration
+      </h2>
+      <p>
+        As this specification is reliant on <a>URLs</a>, implementers need to
```

Comment:
They are never exposed to the user, presumedly (but I couldn't be 100% sure it would never happen - but people do crazy things). I was going to make a note of that.